sdk/tests/deqp/modules/shared/glsFboCompletenessTests.js

```diff
@@ -737,7 +737,7 @@ goog.scope(function() {
                     testFailedOptions('Framebuffer checked as incomplete, but with wrong status', true);
                 }
             } else if (fboStatus != gl.FRAMEBUFFER_COMPLETE && statuses.isFBOStatusValid(gl.FRAMEBUFFER_COMPLETE)) {
-                testFailedOptions('Framebuffer object could have checked as complete but did not.', true);
+                testPassedOptions('Warning: framebuffer object could have checked as complete but did not.', true);
```

Comment:
Looks good, but would you consider using checkMessage (sdk/tests/js/js-test-pre.js) so that a warning is more visibly issued? At your discretion of course. Not sure about which primitives (checkMessage, testPassed, etc.) are best for this purpose.